server/graphql/v1/types.js

```diff
@@ -576,6 +577,19 @@ export const ExpenseType = new GraphQLObjectType({
           });
         },
       },
+      userTaxFormRequiredBeforePayment: {
+        type: GraphQLBoolean,
+        async resolve(expense) {
+          const incurredYear = moment(expense.incurredAt).year();
+
+          return isUserTaxFormRequiredBeforePayment({
```

Comment:
> Not a blocker for me if this is just for the host dashboard or for "show expense" page (https://opencollective.com/______/expenses/7988)

Yes, this would only be used in those cases. It doesn't make sense to tell the public about tax form status on expenses.

My intuition is that the front end should only query for this information when it really needs it. 

Plus I just looked at the loader file for the first time and I'm not sure exactly what to do in there.

So, are you happy to leave this as is and we just make sure the frontend only queries this when needed?

Reply:
The purpose of `loaders` is to batch the queries. Instead of querying the DB for each entry (in this case, each expense) the goal is to make one big query where you fetch all the data and to do the reconciliation in JS.

But yes, if this is only going to be fetched in the host dashboard then I'm ok with this.